client/src/components/Review/Input.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const Input = props => {
+  //console.log(props.value);
+  return (
+    <div>
+      <label for={props.name}>
```

Comment:
I believe in react you don't use the `for` prop, instead you would use the `htmlfor` because `for` is a reserved word in JavaScript (see for loop). https://reactjs.org/docs/dom-elements.html#htmlfor

Reply:
Updated. Looked up the why and saw that, this was just the rule.